.github/ISSUE_TEMPLATE/documentation.yaml

```
@@ -0,0 +1,40 @@
+name: 📚Documentation
+description: A request to update or improve Euphony documentation
+title: "[documentation] <title>"
+labels: "documentation"
+
+body:
+- type: input
+  attributes:
+    label: Title
+    description: title of documentation
+  validations:
+    required: true
+
+- type: input
+  attributes:
+    label: Does the document's page already exist? 
+    description: Please link to it.
```

last time, yaml syntax error using () without double quote.
Sorry, I didn't catch the syntax error.

```suggestion
    description: "Please link to it. (Ex: https://github.com/euphony-io/euphony/blob/master/CONTRIBUTING.md)"
```